### DISCUSSION TITLE
10 APIs grátis e legais para você consumir

### DISCUSSION THREAD

**Discussion by belmirofss** [frontendbr/forum#2093] [Dicas]
## 10 APIs grátis e legais para você consumir

Tá de bobeira aí em casa querendo fazer alguma coisa de diferente?

[Compilei em um artigo no Medium algumas APIs legais para você se divertir](https://medium.com/reactbrasil/10-apis-gr%C3%A1tis-e-legais-para-voc%C3%AA-consumir-69141988ea0b) e estou aqui divulgando para caso alguém tenha interesse e ver.

**Comment by lucalves** [comment]
Um repositório com várias api's interessantes também é o `public-apis`, pode ser encontrado [aqui](https://github.com/public-apis/public-apis).

Seu artigo também ficou muito bom.

**Reply by mlumoura** [reply]
Muito útil! Obrigada!

**Reply by luisotavio-dev** [reply]
Opa, criei um Workspace com APIs úteis como essas, mas adicionando várias nacionais 🇧🇷:
https://www.postman.com/lively-meteor-537630/workspace/useful-apis

**Comment by GabrielScalici** [comment]
Bem legal seu artigo, irmão!

**Comment by DiegoEmanuel** [comment]
valeu

**Comment by ghost** [comment]
Daora o artigo.

**Comment by vitoneto** [comment]
Achei genial, na verdade eu achei o artigo pesquisando no google mesmo. Vou brincar com o do Rick and Morty. Muito brabo, vlw.

**Comment by andersonvaler** [comment]
Nessa lista de apis grátis e legais não pode faltar a Brasil api

https://brasilapi.com.br

**Comment by EdilandoSaturnino** [comment]
Valeu meu chapa, estava precisando, irei utilizá-las com prazer!

**Comment by anaisegomes** [comment]
Obrigada (y)

**Comment by gabrielanacletoo455** [comment]
obrigado amigo, você é um amigo

**Comment by Chikenito** [comment]
Vlw meu camarada, vou utilizar com certeza.
